internal/db/schema/migrations/oss/postgres/78/01_deleted_tables_and_triggers.up.sql

```diff
@@ -2,97 +2,176 @@
 -- SPDX-License-Identifier: BUSL-1.1
 
 begin;
-  create table target_deleted (
+  create table target_tcp_deleted (
     public_id wt_public_id primary key,
     delete_time wt_timestamp not null
   );
 
+  create table target_ssh_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp not null
+  );
+
+  -- Target repository list logic is implemented
+  -- against views, so use a view for the deleted
+  -- ids too.
+  -- Note: does not end in _deleted so as
+  -- not to be attached a trigger in the below function.
+  create view target_all_subtypes_deleted_view
+  as
+    select * from target_tcp_deleted
+    union
+    select * from target_ssh_deleted;
```

Comment:
```suggestion
    select public_id, delete_time from target_tcp_deleted
    union
    select public_id, delete_time from target_ssh_deleted;
```
Using a `select *` in a view can have side effects when an underlying table is updated.